packages/material-ui-lab/package.json

```diff
@@ -33,15 +33,15 @@
     "version": "yarn build:copy-files && npm publish build"
   },
   "peerDependencies": {
-    "material-ui": "^1.0.0-beta.33",
     "@material-ui/icons": "^1.0.0-beta.17",
```

Comment:
Did you forget to actually jump the version?

Reply:
I left it to what it was changed to when the icons package was scoped, as it's covered by the existing range. It just hadn't been released.

Reply:
Why removing the peer dependency?